hocr-eval

```diff
@@ -132,18 +141,12 @@ if(imgfile):
     draw=ImageDraw.Draw(im)
 
 # get pages from inputs
-gtfile = codecs.open(args[0],encoding='utf-8')
-truth_doc = gtfile.read()
-gtfile.close()
-ocrfile = codecs.open(args[1],encoding='utf-8')
-actual_doc = ocrfile.read()
-ocrfile.close()
-
-# parse pages using beautiful soup
-gtsoup = BeautifulSoup(truth_doc,convertEntities='html')
-truth_pages = gtsoup('div','ocr_page')
-ocrsoup = BeautifulSoup(actual_doc,convertEntities='html')
-actual_pages = ocrsoup('div','ocr_page')
+truth_doc = html.fromstring(args[0])
+actual_doc = html.fromstring(args[0])
```

Comment:
Yes, indeed. Thank you for correcting the mistake. This should be ready now, right?